PokemonRampUp/Model/PokemonJSON.swift

```diff
@@ -7,13 +7,30 @@
 
 import Foundation
 
+// Protcol to insure that a class has a init block that takes no parameters
+protocol Initializable {
+    init()
+}
+
 // PokemonJSON struct represents a Pokemon object that is decodbale as this is a responce model
-struct PokemonJSON: Decodable {
+struct PokemonJSON: Decodable & Initializable {
     let name: String
     let elementTypes: [ElementType]
     let abilities: [Ability]
     let moves: [Move]
     let sprites: Sprites
+    
+    // Created this init block for testing. If this init block is called that will create a pichu pokemon object.
```

Comment:
If you do `PokemonJSON()` it will create a pichu PokemonJSON object. This was done to make testing eaiser.